src/callbacks_step/analysis_surface_2d.jl

```diff
@@ -0,0 +1,277 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+# This file contains callbacks that are performed on the surface like computation of
+# surface forces
+
+"""
+    AnalysisSurface{Semidiscretization, Variable}(semi,
+                                                          boundary_symbol_or_boundary_symbols,
+                                                          variable)
+
+This struct is used to compute pointwise surface values of a quantity of interest `variable`
+alongside the boundary/boundaries associated with particular name(s) given in `boundary_symbol`
+or `boundary_symbols`.
+For instance, this can be used to compute the surface pressure coefficient [`SurfacePressureCoefficient`](@ref) or
+surface friction coefficient [`SurfaceFrictionCoefficient`](@ref) of e.g. an airfoil with the boundary
+name `:Airfoil` in 2D.
```

Comment:
```suggestion
symbol `:Airfoil` in 2D.
```

We should clarify this also here:

https://github.com/trixi-framework/Trixi.jl/blob/655497374d9991a7bd814b08af6751e81becab7d/src/callbacks_step/analysis_surface_integral_2d.jl#L21

Reply:
Doesn't it still say `name`?